### DISCUSSION TITLE
a fresh hand's confussion: cellranger-atat peaks or MACS2 peaks

### DISCUSSION THREAD

**Discussion by 0106WeiWeiDeng** [stuart-lab/signac#1022] [Q&A] [answered]
hello, eveyone, I am new to the field of scATAC-seq data processing, and now I have a confussioin. In the Signac vignette of PBMC, the tutorial used cellranger-atac peaks file as input from the beginning. However,  in the paper "Single-cell chromatin state analysis with Signac", authors sugguested that "This revealed a bias in Cellranger for aberrant merging of multiple distinct peaks into a single region" in section "Analysis of multimodal human PBMC data". So, my confussion is that should I use MACS2 peaks as input from the beginning and How?

**Comment by timoast** [comment, marked as answer]
For the tutorials we start with the output of 10x cellranger for simplicity, but I would recommend exploring different peak calling methods if you want to get optimal results for things like motif enrichment.

If you want to use MACS2 peaks, you can see the [peak calling vignette](https://satijalab.org/signac/articles/peak_calling.html) for examples of how to call peaks. You would then run `FeatureMatrix()` using the new set of peaks to generate a count matrix, which you can use as input to create a new assay and follow the vignettes shown on the Signac website.

In cellranger-atac v2 they also improved the peak calling method, so the newer release of cellranger may give better results than what was shown in the Signac paper.
